doc/source/rllib/rllib-connector.rst

```diff
@@ -0,0 +1,272 @@
+.. include:: /_includes/rllib/announcement.rst
+
+.. include:: /_includes/rllib/we_are_hiring.rst
+
+Connectors (Alpha)
+==================
+
+Connector are components that handle transformations on inputs and outputs of a given RL policy, with the goal of improving
+the durabilty and maintainability of RLlib's policy checkpoints.
+
+RLlib algorithms usually require one or more *user environments* and *policies* (usually a neural network).
+
+Data observed from the environments usually go through multiple steps of preprocessing before they reach
+the policy, while the output of the policy also gets transformed multiple times before they are used to control
+specific agents in the environments.
+
+By consolidating these transformations under the framework of connectors, users of RLlib will be able to:
+
+- Restore and deploy individual RLlib policies without having to restore training related logics of RLlib Algorithms.
+- Ensure policies are more durable than the algorithms they get trained with.
+- Allow policies to be adapted to work with different versions of an environment.
+- Run inference with RLlib polcies without worrying about the exact trajectory view requriements or state inputs.
+
+Connectors can be enabled by setting ``enable_connectors`` parameter to ``True``.
```

Comment:
... by setting _the_ ``enable_connectors`` parameter to ``True``.

Reply:
done